CHANGES.rst

```diff
@@ -368,7 +368,10 @@ astropy.utils
 - ``diff_values()``, ``report_diff_values()``, and ``where_not_allclose()``
   utility functions are moved from ``astropy.io.fits.diff``. [#7444]
 
-astropy.visualization
+- ``invalidate_caches()`` has been removed from the
+  ``astropy.utils.compat`` namespace, use it directly from ``importlib``. [#7872]
+
+  astropy.visualization
 ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Can you fix the syntax for the title here?

Reply:
oops